rpc-client-api/src/custom_error.rs

```diff
@@ -228,6 +231,14 @@ impl From<RpcCustomError> for Error {
                     rewards_complete_block_height,
                 })),
             },
+            RpcCustomError::SlotNotEpochBoundary { slot } => Self {
+                code: ErrorCode::ServerError(JSON_RPC_SERVER_ERROR_SLOT_NOT_EPOCH_BOUNDARY),
+                message: format!(
+                    "Rewards cannot be found because slot {slot} is not the epoch boundary. This \
+                     may be due to gap in ledger or long-term storage"
```

Comment:
```suggestion
                    "Rewards cannot be found because slot {slot} is not the epoch boundary. This \
                     may be due to gap in the queried node's local ledger or long-term storage"
```